include/wil/resource.h

```diff
@@ -5241,13 +5241,19 @@ namespace wil
     {
         unique_hglobal_locked() = delete;
 
-        unique_hglobal_locked(_In_ STGMEDIUM& medium) : unique_any<void*, decltype(&::GlobalUnlock), ::GlobalUnlock>(medium.hGlobal)
+        unique_hglobal_locked(HGLOBAL global) : unique_any<void*, decltype(&::GlobalUnlock), ::GlobalUnlock>(global)

```

Comment:
Should this and the other ctors be marked `explicit` ?

Reply:
OK, I have added explicit to all three constructors.